docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1716,6 +1716,9 @@ See <<disk-based-shard-allocation>> for more information about disk watermarks a
 
 `io_stats` (Linux only)::
 (objects) Contains I/O statistics for the node.
+
+NOTE: These statistics are derived from the `/proc/diskstats` kernel interface.
+This interface accounts for IO performed by all processes on the system, even if you are running {es} within a container.
```

Comment:
Nit: I believe we're supposed to try and wrap the docs source at the 80th column. Not that we're very consistent about this, but still...

```suggestion
This interface accounts for IO performed by all processes on the system, even
if you are running {es} within a container.
```